extensions/amp-video/0.1/amp-video.js

```diff
@@ -491,6 +509,27 @@ class AmpVideo extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Android will show a blank frame between the poster and the first frame in
+   * some cases. In these cases, the video element is transparent. By setting
+   * a poster layer underneath, the poster is still shown while the first frame
+   * buffers, so no FOUC.
+   * @private
+   */
+  createPosterForAndroidBug_() {
+    if (!Services.platformFor(this.win).isAndroid()) {
+      return;
+    }
+    const {element} = this;
+    if (element.querySelector('i-amphtml-poster')) {
+      return;
+    }
+    const poster = createPosterImage(element, element.getAttribute('poster'));
+    poster.classList.add('i-amphtml-android-poster-bug');
```

Comment:
Is this css class already defined somewhere? I don't see a CSS change in this PR (just to double check).

Reply:
Not used for CSS but only for intent (see `docking.js`--usage was wrong, fixed now 👍 )